pkg/api/storage/storage.go

```diff
@@ -38,6 +38,28 @@ func (c *Client) ListBucket(ctx context.Context, opts *contracts.ListOptions) (*
 	return resp, nil
 }
 
+func (c *Client) DeleteBucket(ctx context.Context, name string) error {
+	logger.Debug("Delete bucket")
+	_, httpResp, err := c.apiClient.StorageAPI.
+		StorageApiBucketsDestroy(ctx, name).Execute()
+	if err != nil {
+		logger.Error("Error while listing buckets", zap.Error(err))
+		return utils.ErrorPerStatusCode(httpResp, err)
+	}
+	return nil
+}
+
+func (c *Client) UpdateBucket(ctx context.Context, name string) error {
+	logger.Debug("Updating bucket")
+	_, httpResp, err := c.apiClient.StorageAPI.
+		StorageApiBucketsPartialUpdate(ctx, name).Execute()
+	if err != nil {
+		logger.Debug("Error while creating the project Bucket", zap.Error(err))
```

Comment:
Same question here, is it `creating` or `updating` error? 🤔

Reply:
thanks, the message is wrong